app/models/hardware.rb

```diff
@@ -133,7 +133,7 @@ def provisioned_storage
     if has_attribute?("provisioned_storage")
       self["provisioned_storage"]
     else
-      allocated_disk_storage.to_i + ram_size_in_bytes
+      allocated_disk_storage.to_i
```

Comment:
ok, I am reading: `allocated_disk_storage` == `provisioned_storage`

delete this method and the `virtual_attribute :allocated_disk_storage` below